superset-installer/etc/reflex-provisioner/roles/raf/superset/kerberos/files/config/default/krb5.conf

```diff
@@ -0,0 +1,26 @@
+
+[libdefaults]
+  renew_lifetime = 7d
+  forwardable = true
+  default_realm = GVS.GGN
+  ticket_lifetime = 24h
+  dns_lookup_realm = false
+  dns_lookup_kdc = false
+  default_ccache_name = /tmp/krb5cc_%{uid}
+  #default_tgs_enctypes = aes des3-cbc-sha1 rc4 des-cbc-md5
+  #default_tkt_enctypes = aes des3-cbc-sha1 rc4 des-cbc-md5
+
+[domain_realm]
+  GVS.GGN = GVS.GGN
+
+[logging]
+  default = FILE:/var/log/krb5kdc.log
+  admin_server = FILE:/var/log/kadmind.log
+  kdc = FILE:/var/log/krb5kdc.log
+
+[realms]
+  GVS.GGN = {
+    admin_server = kdc-devopslabs.gvs.ggn
+    kdc = kdc-devopslabs.gvs.ggn
+  }
```

Comment:
How can we change this?

Reply:
krb5 config needs to be on the host machine to whichever cluster it has to make the kerberos connection